src/promptflow/promptflow/executor/_line_execution_process_pool.py

```diff
@@ -326,9 +389,18 @@ def create_executor_fork(*, flow_executor: FlowExecutor, storage: AbstractRunSto
 def exec_line_for_queue(executor_creation_func, input_queue: Queue, output_queue: Queue):
     run_storage = QueueRunStorage(output_queue)
     executor: FlowExecutor = executor_creation_func(storage=run_storage)
+
+    # Wait for the start signal message
+    start_msg = input_queue.get()
+    logger.info(f"Process {os.getpid()} received start signal message: {start_msg}")
+
+    # Send a ready signal message
+    output_queue.put("ready")
```

Comment:
Who receives this "ready" signal?

Reply:
LineProcessManager.start_new_process
`try:
            # Wait for subprocess send a ready message.
            ready_msg = output_queue.get(timeout=30)
            logger.info(f"Process {process.pid} get ready_msg: {ready_msg}")
            self.is_ready = True`

Reply:
fixed